mcprag/server.py

```
@@ -377,15 +380,15 @@
         # Initialize cache manager
         if CACHE_SUPPORT:
             self.cache_manager = CacheManager(
-                ttl=Config.CACHE_TTL_SECONDS, max_size=Config.CACHE_MAX_ENTRIES
+                ttl=config.cache_ttl_seconds, max_size=config.cache_max_entries
```

Ruff (0.12.2)

Undefined name `config`

(F821)